### PR TITLE
Adding IDSA Testbed as a Deployment Example

### PR DESCRIPTION
**Pull request by HeinrichPet**

### CODE REVIEW

**Comment by tmberthold** [comment]
The referenced versions in the testbed repo could be outdated compared to the actual running setup, actually two Dataspace Connectors v6.2.0 are used, see [User-Guide which is currently on a PR work in progress](https://github.com/International-Data-Spaces-Association/IDS-testbed/blob/a35570763e76df8b7a2098233fa5d999754093bd/Testbed/Example_ReferenceTestBed_UserGuide) -> "Steps from Connector B to negotiate data from Connector A" -> "self-description of Connector A".

**Comment by HeinrichPet** [comment]
You are right. We have to clearify which version is really used in the testbed...

File: DSC+DAPS+Broker/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#IDSA Testbed
+
+The [IDSA Testbed](https://github.com/International-Data-Spaces-Association/IDS-testbed) provides a setup to test your own Connector implementation with a local IDS deployment. The setup contains two [Dataspace Connectors](https://github.com/International-Data-Spaces-Association/DataspaceConnector), a CA, an [Omjedn DAPS](https://github.com/Fraunhofer-AISEC/omejdn-server) and a [Metadata broker](https://github.com/International-Data-Spaces-Association/metadata-broker-open-core).
```
Comment (brianjahnke) [inline DSC+DAPS+Broker/README.md:3]: Maybe write out "CA"?

**Review by brianjahnke — COMMENTED** [review]
Maybe changing the name from "DSC+DAPS+Broker" to something like "ids-stack" could help? When the setup changes the name gets invalidated and maybe not everyone knows what the abbreviations stand for.

**Review by brianjahnke — APPROVED** [review]

**Pull request merged**